packages/eslint-config-javascript/index.js

```diff
@@ -358,6 +358,26 @@ module.exports = {
             anonymous: 'always',
             named: 'never',
             asyncArrow: 'always'
-        }]
+        }],
+
+        /*
+            The following are rules that would be enabled or the configuration changed as part of the upgrade to ESLint
```

Comment:
Could you file an issue reminding us to evaluate these and enable the ones we agree with?

Reply:
#112